lib/prism/node_ext.rb

```diff
@@ -52,4 +52,48 @@ def options
       o
     end
   end
+
+  class ConstantReadNode < Node
```

Comment:
In your original suggestion, both this class and the one below where `ConstantPathNode`. I figured this one was supposed to be `ConstantReadNode`. Let me know if that's not right.

Reply:
Perfect